docs/howtos/DataPlane Generation - DPG/04renaming.mdx

```diff
@@ -177,7 +215,26 @@ client.get_computed_model()
 <TabItem value="csharp" label="CSharp" >
 
 ```csharp
-FIXME
+namespace PetStoreNamespace.Models
+{
+  public partial class InputModel
+  {
+    public string Name { get; }
+  }
+}
+
+namespace PetStoreNamespace
+{
+  public partial class PetStoreNamespaceClient
+  {
+    // protocol method
+    public virtual async Task<Response> ReadModelAsync(string name, RequestContext context)
+    public virtual Response ReadModel(string name, RequestContext context)
+    // convenience method
+    public virtual async Task<Response<InputModel>> ReadModelAsync(string name, CancellationToken cancellationToken = default)
+    public virtual Response<InputModel> ReadModel(string name, CancellationToken cancellationToken = default)
```

Comment:
let add the curly braces to these methods like this:
```suggestion
    public virtual async Task<Response> ReadModelAsync(string name, RequestContext context) {}
    public virtual Response ReadModel(string name, RequestContext context) {}
    // convenience method
    public virtual async Task<Response<InputModel>> ReadModelAsync(string name, CancellationToken cancellationToken = default) {}
    public virtual Response<InputModel> ReadModel(string name, CancellationToken cancellationToken = default) {}
```

Reply:
similar to other methods in this doc

Reply:
@ArcturusZhang resolved